mesheryctl/internal/cli/root/perf/apply.go

```diff
@@ -67,13 +71,29 @@ mesheryctl perf apply meshery-profile --flags
 // Execute a Performance test with creating a new performance profile
 mesheryctl perf apply meshery-profile-new --url "https://google.com"
 
+// Execute a performance profile using the certificate present in the profile
+mesheryctl perf apply meshery-profile --url "https://google.com" --useCert
+
+// Execute a Performance test creating a new performance profile and pass certificate to be used 
+mesheryctl perf apply meshery-profile-new --url "https://google.com" --certPath path/to/cert.pem --useCert
```

Comment:
I think the one flag `--certPath` is enough, what is the purpose of `--useCert`?

Reply:
@gyohuangxin Currently the API supports running a test with/without the certificate contained in a performance profile. `--useCert` is a boolean flag to toggle this behaviour.

`--certPath` is used to add a certificate to a new profile being created. A certificate can be added to a test profile only while creating it. Currently server-side there isn't a way to include a certificate for a single run.

I was initially considering having only one flag that does both of the above behaviours. But the CLI library doesn't permit a flag to have both a boolean behaviour and an argument behaviour. Hence I've broken it down into 2 flags.

Hence 
```
mesheryctl perf apply meshery-profile-new --url "https://google.com" --certPath path/to/cert.pem --useCert

```
causes a new profile to be created with the specified certificate and `--useCert` causes the certificate to be used when the test is run.

Reply:
Understood about adding "--useCert" to allow the user to choose whether to use a certificate in the performance profile. But can we replace it with "--disableCert" and set its default value to false? Because I think it may be confusing that I need to add `--useCert` when I have `--certPath` set. If we use `--disableCert`, we don't need to explicitly add another flag and keep the command concise, like
```
mesheryctl perf apply meshery-profile-new --url "https://google.com" --certPath path/to/cert.pem 
```
If someone doesn't want to use the certificates in a proformance profile, then he can use `--disableCert`
```
mesheryctl perf apply meshery-profile --url "https://google.com" --disableCert
```
What do you think?

Reply:
@gyohuangxin I like the idea having certificates being used by default. Sure I'll change the flag to `--disableCert`.